docs/scaling/multi-node-setup.txt

```diff
@@ -111,62 +120,62 @@ for more information.
 Master Node Election
 ====================
 
-Although all CrateDB nodes in a cluster are equal, one node is elected as
-master for managing cluster meta data. Like any other peer-to-peer system,
-nodes communicate with each other directly. The master node is responsible for
-making changes to, publishing the global cluster state, and delegating
-redistribution of shards when nodes join or leave the cluster. All nodes are
-eligible to become master.
+In a CrateDB cluster only one master node is responsible for making changes to the
+global cluster state. The master node is elected as part of the
+:ref:`shared_nothing_master_node_election`.
 
-There may only be one master per cluster. To ensure this, CrateDB sets a quorum
-which needs to be present to elect a master and for the cluster to be
-operational.
+. NOTE::
+   The following setting only affects versions <= 4.x of CrateDB.
 
-This quorum can be configured using the ``minimum_master_nodes`` setting. It's
-highly recommend to set the quorum greater than half the maximum number of
-nodes in the cluster:
+To ensure this, CrateDB sets a quorum which needs to be present to elect a master and
```

Comment:
"To ensure this, CrateDB sets a quorum which needs to be present to elect a master and for the cluster to be operational" 
====> 
"To ensure this, CrateDB configures a quorum of nodes which need to be present to elect a master and participate in future master elections, keeping the cluster operational"